tests/driver_sx127x/main.c

```diff
@@ -302,10 +305,10 @@ static void _event_cb(netdev_t *dev, netdev_event_t event)
             case NETDEV_EVENT_RX_COMPLETE:
```

Comment:
This application is not part of gnrc_netdev. It's just here to show how the underlying thread communicate with the driver netdev interface.
gnrc_netdev has it's own thread [here](https://github.com/RIOT-OS/RIOT/blob/master/sys/net/gnrc/link_layer/netdev/gnrc_netdev.c#L49) and the gnrc_netdev adaption of sx127x uses this one now, via the auto_init stuff.